src/asmcnc/apps/drywall_cutter_app/config/config_loader.py

```diff
@@ -98,6 +99,29 @@ def get_shape_dimensions(json_obj):
     return dims
 
 
+def migrate_10_to_20(config, file_path):
```

Comment:
Without context this name is quite misleading with '10' and '20'.

Reply:
Yeah I know but since you can't have "1.0" in a function name I thought this and the comment would be good enough. I'm open to suggestions though

Reply:
v1_to_v2?